Tchap/Modules/Favourites/FavouriteMessages/FavouriteMessagesViewController.swift

```diff
@@ -61,11 +61,8 @@ final class FavouriteMessagesViewController: UIViewController {
         super.viewDidLoad()
         
         // Do any additional setup after loading the view.
-        // Remove back bar button title when pushing a view controller
-        self.navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
```

Comment:
Are you sure you want to remove this code?
We are used to use it to remove text on the < (back chevron)
I think  you should restore it

Reply:
This code seems useless at least on iOS 14
We will test later on iOS11

This remark can be ignored for the moment